openprescribing/templates/measure_for_one_entity.html

```diff
@@ -10,22 +10,35 @@
 {% endblock extra_css %}
 
 {% block content %}
+  {% if entity %}
+
   <h1>
     {{ measure.name }}
     <br><small>{{ entity.name }}</small>
   </h1>
-  {% if entity_type == 'practice' %}
-    <p class="text-muted" style="margin-top: -8px">
-      {{ entity.address_pretty_minus_firstline }}
+  {% if entity %}
+    {% if entity_type == 'practice' %}
+      <p class="text-muted" style="margin-top: -8px">
+        {{ entity.address_pretty_minus_firstline }}
+      </p>
+    {% endif %}
+    <p>
+      <a href="{% url measures_url_name entity.code %}" class="btn btn-default">
+        View all measures for this {{ entity_type_human }} &rarr;
+      </a>
     </p>
-  {% endif %}
-  <p>
-    <a href="{% url measures_url_name entity.code %}" class="btn btn-default">
-      View all measures for this {{ entity_type_human }} &rarr;
-    </a>
-  </p>
 
-  <p>This measure shows how this organisation compares with its peers across NHS England. This is indicative only, and should be approached with caution. <a href='{% url 'faq' %}#measureinterpret'>Read more about measures</a>.</p>
+    <p>This measure shows how this organisation compares with its
+    peers across NHS England. This is indicative only, and should be
+    approached with caution.
+
+    <a href='{% url 'faq' %}#measureinterpret'>Read more about measures</a>.</p>
+
+  {% else %}
+
+  <h1>{{ measure.name }}</h1>
```

Comment:
The `else` block and the main body of the conditional above, could both do with indenting I think.